Dockerfile

```diff
@@ -3,10 +3,15 @@ FROM node:14-buster-slim as build-frontend
 COPY package.json ./
 COPY yarn.lock ./
 
+RUN apt update && \
+    apt upgrade -y
+
+RUN apt-get install -y build-essential python
+
 WORKDIR /app
 
 COPY ./app/package.json .
-RUN yarn install
+RUN yarn install --network-timeout 600000
```

Comment:
Is the timeout actually needed? :thinking: 
To speed up `yarn install` we could also use the following flags:

* --non-interactive
* --frozen-lockfile
* --prefer-offline

Reply:
The timeout doesn't slow anything down. It will just slow everything down in a worst case scenario where the pipeline will fail eventually. But because the install is very slow, a timeout bigger than the default is needed. But i will test if i can reduce it.

Reply:
The reduced timeout is enough. Because the build process isn't very stable, i would suggest to not try to optimize it but to be thankfull that it works. :grin:

Reply:
We should at least add `--non-interactive`.

Reply:
It doesn't fail sometimes, but depending on which architecture and js dependencies you install, you need to fiddle with different settings. Right now its stable.

Reply:
But i will test non interactive

Reply:
It seems to work. MR is ready to merge in my opinion.

Reply:
We should also add these flags to all other `yarn` calls.
But let's to this in an extra PR.